storops/vnx/resource/system.py

```diff
@@ -116,6 +116,20 @@ def _init_file_cli(self):
                             self._file_username,

```

Comment:
OK.  I don't think we need to clear the queue.  That probably means we don't have other cleanups to do on `VNXSystem`.  Thanks.